packages/docusaurus-theme-classic/codeTranslations/base.json

```diff
@@ -4,9 +4,9 @@
   "theme.CodeBlock.copied": "Copied",
   "theme.CodeBlock.copied___DESCRIPTION": "The copied button label on code blocks",
   "theme.CodeBlock.copy": "Copy",
+  "theme.CodeBlock.copy___DESCRIPTION": "The copy button label on code blocks",
   "theme.CodeBlock.copyButtonAriaLabel": "Copy code to clipboard",
   "theme.CodeBlock.copyButtonAriaLabel___DESCRIPTION": "The ARIA label for copy code blocks button",
-  "theme.CodeBlock.copy___DESCRIPTION": "The copy button label on code blocks",
```

Comment:
base is auto-generated so you should not update it, it will be reverted anyway

Will see how to update the auto-gen script to fix this, thanks!